package NYT.NHiveClient.NProto;

import "yt_proto/yt/core/misc/proto/guid.proto";
import "yt_proto/yt/client/node_tracker_client/proto/node.proto";

////////////////////////////////////////////////////////////////////////////////

message TCellPeerDescriptor
{
    required NYT.NNodeTrackerClient.NProto.TNodeDescriptor node_descriptor = 1;
    required bool voting = 2;
    optional string alien_cluster = 3;
}

message TCellDescriptor
{
    required NYT.NProto.TGuid cell_id = 1;
    required int32 config_version = 2;
    repeated TCellPeerDescriptor peers = 3;
}

message TCellInfo
{
    required NYT.NProto.TGuid cell_id = 1;
    required int32 config_version = 2;
}

////////////////////////////////////////////////////////////////////////////////
